pylint_odoo/misc.py

```diff
@@ -112,14 +114,41 @@ def get_manifest_file(self, node_file):
 
     def set_ext_files(self):
         """Create `self.ext_files` dictionary with {extension_file: [files]}
+            and exclude files using --ignore and --ignore-patterns parameters
         """
         self.ext_files = {}
+        base_path = os.path.dirname(os.path.realpath(__file__))
+        ignores = []
+        ignore_argvs = getopt.getopt([argv for argv in sys.argv[1:]
```

Comment:
The parameters are received from `config` like as other custom parameters.
Because a configfile could be used too.

Search where is used the `config` variable to read by example the js configuration file for js-lint check